myuw/test/dao/myuw_notice.py

```diff
@@ -74,38 +81,60 @@ def test_affil(self):
                             content="Notice Content Four",
                             notice_type="Banner",
                             notice_category="Student",
-                            start=datetime(2018, 6, 8, 10, 0, 0),
-                            end=datetime(2018, 6, 20, 10, 0, 0),
+                            start=get_datetime_with_tz(2018, 6, 8, 10),
+                            end=get_datetime_with_tz(2018, 6, 20, 10),
                             is_instructor=True)
         notice.save()
-        notice = MyuwNotice(title="Test",
+        notice = MyuwNotice(title="All Student",
                             content="Notice Content Five",
                             notice_type="Banner",
                             notice_category="Student",
-                            start=datetime(2018, 6, 8, 10, 0, 0),
-                            end=datetime(2018, 6, 20, 10, 0, 0),
+                            start=get_datetime_with_tz(2018, 6, 8, 10),
+                            end=get_datetime_with_tz(2018, 6, 20, 10),
                             is_student=True)
         notice.save()
+        notice = MyuwNotice(title="Seattle Intl",
+                            content="Notice Content Five",
+                            notice_type="Banner",
+                            notice_category="Student",
+                            start=get_datetime_with_tz(2018, 6, 8, 10),
+                            end=get_datetime_with_tz(2018, 6, 20, 10),
+                            is_seattle=True,
+                            is_intl_stud=True)
+        notice.save()
         request = get_request_with_date("2018-06-09")
         get_request_with_user('javerage', request)
         notices = get_myuw_notices_for_user(request)
         self.assertEqual(len(notices), 1)
```

Comment:
Stephen, without the change on line 48 above, this assertion on line 108 will fail due to the lenngth being 2.